sdks/go/pkg/beam/core/util/reflectx/functions_test.go

```diff
@@ -41,3 +45,19 @@ func TestLoadFunction(t *testing.T) {
 		t.Errorf("got %d, wanted %d", out[0].Int(), testFunction())
 	}
 }
+
+func TestFunctionOutputSize(t *testing.T) {
+	expected := 1
+	received := FunctionOutputSize(testFunction)
+	if received != expected {
+		t.Errorf("got %d, wanted %d", received, expected)
+	}
+}
+
+func TestFunction2OutputSize(t *testing.T) {
+	expected := 2
+	received := FunctionOutputSize(testFunction2)
```

Comment:
Idiomatically, Go uses "got and want" as the variables, as well as in the test error output.

Reply:
a4d5083